### PR TITLE
Highlight points of complexity

### PR DESCRIPTION
**Pull request by brentzundel**
This PR introduces text which highlights some of the complexity that may come from using DIDs, in addition to the outlined benefits of DIDs, to paint a more complete picture.
This PR addresses issue #36

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -153,6 +153,35 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 (including, but not limited to, switching between Verifiable Credential digital wallet providers).
           </li>
         </ul>
+      <p>
+Conceptually, the relationship of a DID and a DID method is similar to the
```
Comment (TallTed) [inline index.html]: `relationship of a DID`
-> `relationship between a DID`

File: index.html [hunk 2]
```diff
@@ -153,6 +153,35 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
...
+relationship of an IETF generic URI and a specific URI scheme (such as the http:
```
Comment (TallTed) [inline index.html]: `relationship of an IETF`
-> `relationship between an IETF`

File: index.html [hunk 3]
```diff
@@ -153,6 +153,35 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
...
+and https: schemes). It is also similar to the relationship of an IETF generic
```
Comment (TallTed) [inline index.html]: `relationship of an IETF`
-> `relationship between an IETF`

**Review by ken-ebert — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -153,6 +153,35 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 (including, but not limited to, switching between Verifiable Credential digital wallet providers).
           </li>
         </ul>
+      <p>
```
Comment (msporny) [inline index.html:156]: The more text a W3C Charter has, the more likely one of the 430+ W3C Members are to object to one of the sentences. W3C Charters are meant to be extremely sparse (some would say that the DID WG Charter already has way too much in it). 

The text below is good, we should put it in the DID spec. 

Suggesting that "DIDs ... lead to some complexity" will provide something to pounce on for the organizations that don't want to see this work go anywhere.

**Review by msporny — CHANGES_REQUESTED** [review]
Let's move this text into the DID spec and not put it in the DID WG Charter.

**Comment by msporny** [comment]
@brentzundel could you please raise a PR on the DID spec, please? Leaving this PR open until you do that so we don't forget. I suggest putting it in the section about DID Methods.

**Comment by jandrieu** [comment]
I'm with @msporny . Similar to #58, these are details suitable to the spec. Not only for the reasons Manu mention, but also because the language is going to change as we advance the work. What goes in the charter should set clear bounds in terms that can suffice for the entire term of the work.

That said, i like the explanation. Perhaps some of the language here could be absorbed into https://github.com/w3c-ccg/did-spec/pull/234

**Comment by brentzundel** [comment]
I have moved this text to [a PR on the DID Spec](https://github.com/w3c-ccg/did-spec/pull/239)

**Comment by peacekeeper** [comment]
Since https://github.com/w3c-ccg/did-spec/pull/239 has been merged, I think this PR can now be closed.

**Comment by brentzundel** [comment]
Closing, as the related PR has been merged into the spec.

**Pull request closed without merging**